src/level/Tile/WallTile.cpp

```diff
@@ -14,13 +14,15 @@ namespace Tile
 		m_height(height)
 	{}
 
-	void WallTile::render(uint32 x, uint32 y, Level& level, sf::RenderWindow& window)
-	{
-		LevelRenderer::renderWallTileTop(x * TILE_SIZE, y * TILE_SIZE, 32, m_sprite);
-		Tile* nextTile = level.getTile(x, y + 1);
-		if (nextTile == nullptr || !nextTile->isSolid())
+		void WallTile::render(uint32 x, uint32 y, Level& level, sf::RenderWindow& window)
 		{
-			LevelRenderer::renderWallTileSide(x * TILE_SIZE, y * TILE_SIZE, m_spriteSide);
+			LevelRenderer::renderWallTileTop(x * (uint)TILE_SIZE, y * (uint)TILE_SIZE, 32, m_sprite);
+			Tile* nextTile = level.getTile(x, y + 1);
+			if (nextTile == nullptr || !nextTile->isSolid())
+			{
+				LevelRenderer::renderWallTileSide(x * (uint)TILE_SIZE, y * (uint)TILE_SIZE, m_spriteSide);
+			}
 		}
+
 	}
-}}
\ No newline at end of file
+}
```

Comment:
Also, please install editorconfig for your IDE/text editor. It will automatically configure your editor for the rules we specified in `.editorconfig` file.